R/model.R

```diff
@@ -1073,6 +1077,7 @@ optimize <- function(data = NULL,
 
   cmdstan_procs <- CmdStanProcs$new(
     num_procs = 1,
+    iter = ifelse(is.null(optimize_args$iter),1e3,optimize_args$iter),
```

Comment:
```suggestion
    iter = optimize_args$iter %||% 1e3,
```

Reply:
Also is 1e3 the right value to use for optimize?

Reply:
Good catch, it's actually 1e4

Reply:
oops, that's variational. optimize is 2e3